assets/app/styles/_pods.less

```diff
@@ -1,17 +1,101 @@
+// Project overview
+
+.project-overview-page {
```

Comment:
it doesnt really make sense organizationally for this to be in pods.less, maybe in core? or in a file just for overview?